packages/ui/stories/2_components/actionMenu/withActionMenu.stories.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { withActionMenu } from '@tidb-community/ui';
+import { Menu, Dropdown, Button } from 'antd';
+
+import { getTitle } from '../utils';
+
+const ActionMenu = withActionMenu(Menu);
+
+export default {
+  title: getTitle('ActionMenu'),
+  component: ActionMenu,
```

Comment:
If you don't want to control other menu props, you could remove this component from the setting.